<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="T" failures="2" tests="5" name="Submit Order Test" time="25.642" errors="0" timestamp="2024-12-12T20:59:28 IST">
  <testcase classname="Twarita.e2e_Selenium_TestNG_Framework.tests.SubmitOrderTest" name="submitOrder" time="8.289"/>
  <testcase classname="Twarita.e2e_Selenium_TestNG_Framework.tests.SubmitOrderTest" name="submitOrder" time="0.561">
    <skipped/>
  </testcase> <!-- submitOrder -->
  <testcase classname="Twarita.e2e_Selenium_TestNG_Framework.tests.SubmitOrderTest" name="submitOrder" time="0.574">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)&amp;quot; because the return value of &amp;quot;Twarita.e2e_Selenium_TestNG_Framework.pageObjects.ProductCatalogue.getProduct(String)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because the return value of "Twarita.e2e_Selenium_TestNG_Framework.pageObjects.ProductCatalogue.getProduct(String)" is null
at Twarita.e2e_Selenium_TestNG_Framework.pageObjects.ProductCatalogue.addProductToCart(ProductCatalogue.java:36)
at Twarita.e2e_Selenium_TestNG_Framework.tests.SubmitOrderTest.submitOrder(SubmitOrderTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- submitOrder -->
  <testcase classname="Twarita.e2e_Selenium_TestNG_Framework.tests.SubmitOrderTest" name="verifyOrderPlaced" time="0.573"/>
  <testcase classname="Twarita.e2e_Selenium_TestNG_Framework.tests.SubmitOrderTest" name="verifyOrderPlaced" time="0.786">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Twarita.e2e_Selenium_TestNG_Framework.tests.SubmitOrderTest.verifyOrderPlaced(SubmitOrderTest.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- verifyOrderPlaced -->
</testsuite> <!-- Submit Order Test -->
